modules/tls-profiles-ingress-configuring.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-ingress-configuring_{context}"]
+= Configuring TLS profiles for the Ingress controller
+
+To configure a TLS security profile for the Ingress Controllers, edit the `IngressController` custom resource (CR) to specify a predefined or custom TLS profile. If a TLS profile is not configured, the default value is based on the TLS profile set of the API server.
+
+.Sample `IngressController` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: IngressController
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+ ...
+----
+
+The TLS security profile defines the minimum TLS version and the TLS ciphers for TLS connections for Ingress controllers.
+
+You can see the ciphers and the minimum TLS version of the configured TLS security profile in the `IngressController` custom resource (CR) under `Spec.Tls Security Profile` and `Status.Tls Profile`.
+
+[IMPORTANT]
+====
+The HAProxy Ingress controller image does not support TLS `1.3` and because the `Modern` profile requires TLS `1.3` it is not supported. The Ingress Operator converts the `Modern` profile to `Intermediate`.
+
+The Ingress Operator also converts the TLS `1.0` of an `Old` or `Custom` profile to `1.1`, and TLS `1.3` of a `Custom` profile to `1.2`.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the `IngressController` CR in the `openshift-ingress-operator` project to configure the TLS security profile:
++
+----
+$ oc edit IngressController default -n openshift-ingress-operator
+----
++
+.Sample `IngressController` CR for a `Custom` profile
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+ ...
+spec:
+  tlsSecurityProfile:
+    type: Custom <1>
+    custom: <2>
+      ciphers: <3>
+      - ECDHE-ECDSA-CHACHA20-POLY1305
+      - ECDHE-RSA-CHACHA20-POLY1305
+      - ECDHE-RSA-AES128-GCM-SHA256
+      - ECDHE-ECDSA-AES128-GCM-SHA256
+      minTLSVersion: VersionTLS11
+ ...
+----
+<1> Specify the TLS security profile type. The default is the TLS profile of the API server.
+* `Old`
```

Comment:
This is taking up quite a bit of vertical space as bullets. Could you consider condensing on to a single line, like:

```
Specify the TLS security profile type (`Old`, `Intermediate`, or `Custom`). The default is the TLS profile of the API server.
```

I think it makes sense to have bullets for callout 2 though, since those are values that are on their own lines.

Same for the other procedures, if you do make any changes.